e2e/storybook-angular/src/storybook-angular.test.ts

```diff
@@ -15,7 +15,7 @@ describe('Storybook executors for Angular', () => {
   const angularStorybookLib = uniq('test-ui-ng-lib');
   beforeAll(() => {
     newProject();
-    createTestUILib(angularStorybookLib);
+    runCLI(`g @nrwl/angular:library ${angularStorybookLib} --no-interactive`);
```

Comment:
Change this to `@nx/angular` since we are anyways migrating the entire repo to `@nx/*`